### PR TITLE
EarnList and Distribution Query Param

### PR DESCRIPTION
**Pull request by youngkidwarrior**
For now I'm keeping the name "distribution" in the codebase

We should try to maintain parity between the codebase and the marketing. Once we settle on the new name for "distributions", I will update our variable and component names to match

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/176?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#176** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/176?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#169** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/169?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#168** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/168?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#167** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/167?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#161** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/161?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/features/account/earn-tokens/screen.tsx [hunk 1]
```diff
@@ -93,10 +101,65 @@ const DistributionSection = () => {
...
 }
+
+const EarnList = ({ distributions }: { distributions?: UseDistributionsResultData }) => {
+  const { isLoading, error, refetch } = useDistributions()
+  const [distributionNumberParam, setDistributionNumberParam] = useDistributionNumber()
+
+  if (error)
+    return (
+      <Stack w={'100%'} jc="center" ai="center">
+        <Button onPress={() => refetch()}>Reload</Button>
```
Comment (0xBigBoss) [inline packages/app/features/account/earn-tokens/screen.tsx:115]: why not show the error?

**Review by 0xBigBoss — COMMENTED** [review]
Let's include the changes to add the distribution page in this one or the next one.

**Review by 0xBigBoss — APPROVED** [review]

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Mar 14, 3:41 PM EDT**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/176).

**Pull request merged**